### PR TITLE
more redirects

### PR DESCRIPTION
**Pull request by cedrabner**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -283,20 +283,33 @@ const config = {
...
               "/get-started/configure-metamask",
               "/get-started/quickstart",
               "/use-zkevm",
-              "/use-linea/fund",
-              "/use-linea/bridge-funds",
               "/use-linea/index.md",
             ],
           },
+          {
+            to: "/build-on-linea/use-linea-testnet/bridge-funds",
```
Comment (emilyJLin95) [inline docusaurus.config.js:302]: Is this a duplicate of line 310?

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (emilyJLin95) [reply docusaurus.config.js:302]: did you mean to use `get-started`? and then collapse that into one entry with two from links in teh list

Comment (cedrabner) [reply docusaurus.config.js:302]: oh yeah i guess it is a duplicate. i am not sure what get-started section is supposed to link to. oliver made those initial redirects so i didn't want to change what i wasn't sure about. ill get rid of the duplicate one though

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**